### DISCUSSION TITLE
Weather to add "./" to subpart codes?

### DISCUSSION THREAD

**Discussion by An4el** [idaholab/moose#27978] [Q&A Modules: General] [answered]
In the Example 1 : As Simple as it Gets, most of the subpart have a "./" suffix on it and ends with "[../]", like this:
`[Variables]
   [./diffused]
    order = FIRST
    family = LAGRANGE
  [../]
[]`
and in tutorial of Phase Field, there is no syntax like this:
`[Variables]
  [c]
    order = FIRST
    family = LAGRANGE
  []
  [w]
    order = FIRST
    family = LAGRANGE
  []
[]`
So what's the difference between them?

**Comment by GiudGiud** [comment, marked as answer]
Hello

./ and ../ are no longer necessary.

Guillaume
